ant/__init__.py

```diff
@@ -20,4 +20,6 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+from __future__ import absolute_import, print_function
```

Comment:
It may be discussed. It can be removed if unused, but risk is to forget to add an important required `__future__` import in a later commit. Usually, in projets with python 2&3 support, I keep the same `__future__` imports in every files even if unused.

Keeping those imports in each files makes python2 behaves like python3 without having to think about it too much.

I didn't add `unicode_literals` import because it mess things up with the `struct.pack` and `struct.unpack` functions.